Checkpoint MDS login (#10405)
demisto/content: Packs/CheckpointFirewall/ReleaseNotes/2_0_6.md

#### Integrations
##### Check Point Firewall v2

> - Updated ***checkpoint-login-and-get-session-id*** to add optional domain parameter allowingn MDS login

Reviewer: ```suggestion
- Added the optional ***domain*** parameter to allow MDS login.
```
